packages/core/src/combinator/zip.ts

```diff
@@ -29,18 +35,21 @@ export function zip (f, stream1, stream2) {
 * @returns {Stream} new stream with items at corresponding indices combined
 *  using f
 */
-export const zipArray = (f, streams) =>
+export const zipArray = <Args extends any[], R>(f: (...args: Args) => R, streams: ToStreamsArray<Args>): Stream<R> =>
```

Comment:
It makes sense, since `unknown` is the top type (whereas any is _both_ top and bottom).  I remember having some weird problems (i.e. TS bugs?) with `<A extends unknown[]>` in some other projects, but maybe those have been resolved.  I'm ok with either for now.

Reply:
done